$NetBSD: patch-ag,v 1.2 2010/10/19 16:41:31 abs Exp $

# Remove unnecessary byteswap.h as we later include riff.h which includes byteswap.h as needed

--- src/modules/kino/riff.cc.orig	2010-09-14 06:00:09.000000000 +0000
+++ src/modules/kino/riff.cc
@@ -25,9 +25,6 @@
 //#include <stdio.h>
 #include <iostream>
 #include <iomanip>
-#ifndef __FreeBSD__
-#include <byteswap.h>
-#endif /* __FreeBSD__ */
 
 using std::cout;
 using std::hex;
